app/models/in_person_enrollment.rb

```diff
@@ -84,7 +84,7 @@ def generate_unique_id
 
     private
 
-    def is_pending_and_established_between(early_benchmark, late_benchmark)
+    def pending_and_established_between(early_benchmark, late_benchmark)
```

Comment:
This change makes a lot more sense to me and is more in keeping with the spirit. The "is" for something that's not a boolean is confusing; this makes it much clearer. 👍